schema/url/user.raml

```diff
@@ -185,3 +185,43 @@ types:
     additionalProperties: false
     properties:
       count: integer
+
```

Comment:
So to confirm, we don't want to pass the derived variable IDs to the patch endpoints via the descriptor field because that would allow the removal of derived variable IDs?

Reply:
Hmmm... actually maybe this is fine.  Trying to think how much backend validation we should do here.  The compute and viz configs could contain derived vars but we don't/can't really look there.  That was the rationalization for not letting them remove- to keep them from shooting themselves in the foot by removing one they are using.  But maybe that can be the client's problem.  I think as long as long as they can't remove DVs from their user-scoped collection (the derived_vars table), they can't cause damage they/we can't undo.